tests/python/unittest/test_autotvm_xgboost_model.py

```diff
@@ -43,6 +43,8 @@ def test_fit():
 
     upper_model.fit(xs, ys, plan_size=32)
 
+    upper_model.predict(np.ones(12))
```

Comment:
Surprisingly, xgboost seems to be fine doing predictions on vectors longer than what it was trained on.